repobee_sanitizer/_sanitize.py

```diff
@@ -34,6 +34,23 @@ def sanitize_file(file_abs_path: pathlib.Path) -> Optional[str]:
         return "\n".join(sanitized_string)
 
 
+def strip_file(file_abs_path: pathlib.Path) -> Optional[str]:
+    """Runs the strip protocol on a given file. This will remove all sanitizer
+    markers from the file. File must be syntax checked before running this.
+
+    Args:
+        file_abs_path: The absolute file path to the file you wish to
+        sanitize.
+
+    Returns:
+        We return the stripped output text.
```

Comment:
Avoid "we" when writing docstrings. A docstring should say what the _function_ does, and the function is not a we. A returns comment in particular should describe the returned value, and not an action. There are tons of examples of good return comments in [Google's style guide](https://google.github.io/styleguide/pyguide.html#doc-function-returns).

```suggestion
        The stripped output text.
```